src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -42,10 +42,10 @@ $pf-v6-c-tree-view--MaxNesting: 10 !default;
 
   // Toggle icon
   --#{$tree-view}__node-toggle-icon--MinWidth: var(--pf-t--global--icon--size--font--body--default);
-  --#{$tree-view}__node-toggle--PaddingBlockStart: var(--pf-t--global--spacer--sm);
-  --#{$tree-view}__node-toggle--PaddingInlineEnd: var(--pf-t--global--spacer--md);
-  --#{$tree-view}__node-toggle--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
-  --#{$tree-view}__node-toggle--PaddingInlineStart: var(--pf-t--global--spacer--md);
+  --#{$tree-view}__node-toggle--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$tree-view}__node-toggle--PaddingInlineEnd: var(--pf-t--global--spacer--control--horizontal--default);
+  --#{$tree-view}__node-toggle--PaddingBlockEnd: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$tree-view}__node-toggle--PaddingInlineStart: var(--pf-t--global--spacer--control--horizontal--default);
```

Comment:
I don't know that this needs to use a semantic spacer? My guess is the size of this box for the toggle icon just needs to match the tree view node padding. Similar to the table, if someone were to theme the semantic control/action spacer, they probably don't intend for this box to change shape, and if it did,  it could cause the node box to extend out of the parent node.

Reply:
Reverted